src/app/components/AccountMenu/index.tsx

```diff
@@ -73,49 +77,78 @@ function AccountMenu({ showOptions = true }: Props) {
-              {balancesDecorated.accountBalance}
-            </span>
-            {!!balancesDecorated.fiatBalance && (
-              <span className="text-xs text-gray-600 dark:text-neutral-400 ml-2">
-                ~{balancesDecorated.fiatBalance}
-              </span>
-            )}
-          </p>
-        ) : (
-          <Skeleton />
-        )}
-      </div>
-
+    <div className="relative pl-2 flex justify-end w-72">
       <Menu as="div">
-        <Menu.Button className="h-full px-2 rounded-r-md hover:bg-gray-200 dark:hover:bg-white/10 transition-colors duration-200">
-          <CaretDownIcon className="h-4 w-4 dark:text-white" />
-          <span className="sr-only">{t("screen_reader")}</span>
+        <Menu.Button className="h-full px-2 rounded-md hover:bg-gray-200 dark:hover:bg-white/10 transition-colors duration-200">
+          <div className="flex items-center">
+            <Avatar size={24} name={authAccount?.id || ""} />
+            <div
+              className={`flex-auto mx-2 py-3 overflow-hidden min-w-[10rem] max-w-[14rem] text-left`}
+            >
+              <p
+                title={title || ""}
+                className="text-sm font-medium text-gray-700 dark:text-neutral-400 text-ellipsis overflow-hidden whitespace-nowrap"
+              >
+                {title || <Skeleton />}
+              </p>
+            </div>
+            <CaretDownIcon className="h-4 w-4 dark:text-white" />
+            <span className="sr-only">{t("screen_reader")}</span>
+          </div>
         </Menu.Button>
+        <Menu.List position="right" fullWidth>
+          <Menu.Item>
+            <div
+              className={`flex-auto px-4 py-2 overflow-hidden ${
+                !title && !balancesDecorated ? "w-28" : ""
+              }`}
+            >
+              <span className="text-xs text-gray-500 dark:text-neutral-300">
+                {tCommon("balance")}
+              </span>
+              {balancesDecorated.accountBalance ? (
+                <p className="flex justify-between">
+                  <span className="dark:text-white">
+                    {balancesDecorated.accountBalance}
+                  </span>
+                  {!!balancesDecorated.fiatBalance && (
+                    <span className="text-gray-500 dark:text-neutral-300">
+                      ~{balancesDecorated.fiatBalance}
+                    </span>
+                  )}
+                </p>
+              ) : (
+                <Skeleton />
+              )}
+            </div>
+          </Menu.Item>
+          <Menu.ItemButton
+            onClick={() => {
+              openOptions(`accounts/${authAccount?.id}`);
+            }}
+          >
+            <WalletIcon className="h-5 w-5 mr-2 text-gray-700 dark:text-neutral-300" />
+            {t("options.account.account_settings")}
+          </Menu.ItemButton>
+          {isAlbyAccount(authAccount?.alias) && (
+            <Menu.ItemButton
```

Comment:
I am wondering if we should add this also (or only) to the account detail page. 
Right now we say "Account settings" and "manage the Alby account" 
maybe it can only be "Account settings" and in there we have a prominent link to manage the Alby account? 

potentially this could even be its own section explaining the user a bit the difference?